src/classes/VOL_SharedCode.cls

```diff
@@ -374,21 +374,21 @@ global with sharing class VOL_SharedCode {
                     strSoql += ' or ' + str + ' = :strEmail ';
                 }
             }
-            // handle NPSP email fields 
+            // handle NPSP email fields
             if (IsNPSPInstalled) {
                 strSoql += ' or npe01__AlternateEmail__c = :strEmail ';
                 strSoql += ' or npe01__HomeEmail__c = :strEmail ';
                 strSoql += ' or npe01__WorkEmail__c = :strEmail ';
             }
             strSoql += ') ';
-        }        
-        list<Contact> listCon = Database.Query(strSoql); 
+        }
+        list<Contact> listCon = Database.Query(strSoql);
         return listCon.size() > 0;
     }
-    
+
     // global code to lookup an existing contact
     // listStrFields are optional fields to include in the soql call
-    global static list<Contact> LookupContact(Contact contact, list<string> listStrFields) {    
+    global static list<Contact> LookupContact(Contact contactRecord, list<string> listStrFields) {

```

Comment:
I would not hate it if you would fix all the casing on classes throughout. Something about `list<string>` just makes me sad.

Reply:
Also upper case method names ...

Reply:
Actually thank you for not putting too many style changes in the first round of review. But once we're done with the first round if you want a second round to just fix style changes, I'd support it.